src/SAWScript/Crucible/LLVM/Override.hs

```diff
@@ -5,27 +5,26 @@ License     : BSD3
 Maintainer  : atomb
 Stability   : provisional
 -}
-{-# LANGUAGE DataKinds #-}
-{-# LANGUAGE DeriveGeneric #-}
-{-# LANGUAGE FlexibleContexts #-}
-{-# LANGUAGE GADTs #-}
+{-# LANGUAGE DataKinds                  #-}
+{-# LANGUAGE DeriveGeneric              #-}
+{-# LANGUAGE FlexibleContexts           #-}
+{-# LANGUAGE GADTs                      #-}
```

Comment:
This PR contains a fair bit of whitespace reorganization that, while likely useful, appears to be unrelated to the goal of fixing Haddock errors. For the sake of making the `git` history neater, it would be more convenient if these sorts of changes could be put in separate commits.